testserver/testserver.go

```diff
@@ -178,6 +182,27 @@ func SecureOpt() TestServerOpt {
 	}
 }
 
+// StoreOnDiskOpt is a TestServer option that can be passed to NewTestServer
+// to enable storing database in memory.
+func StoreOnDiskOpt() TestServerOpt {
+	return func(args *testServerArgs) {
+		args.storeOnDisk = true
+	}
+}
+
+// SetStoreMemsize is a TestServer option that can be passed to NewTestServer
+// to set the proportion of available memory that is allocated
+// to the test server.
+func SetStoreMemsize(memSize float64) TestServerOpt {
```

Comment:
nit: can this be renamed to `SetStoreMemSizeOpt`

(capital S for Size, and end with Opt)

Reply:
Done.